papaparse.js

```diff
@@ -248,7 +248,7 @@
 			if (_input.data instanceof Array)
 			{
 				if (!_input.fields)
-					_input.fields = _input.data[0] instanceof Array
+					_input.fields = ( _input.meta && _input.meta.fields ) || _input.data[0] instanceof Array
 									? _input.fields
```

Comment:
This needs a little work; if the input is coming from Papa.parse(), then the fields are in `_input.meta.fields`. But we still need to account for the case when the fields are defined in `_input.fields` too. Might require turning this ternary statement into a full `if` and `else` statement or something, to make it easier to read.